src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LoadedProject.cs

```diff
@@ -126,6 +126,11 @@ private void AssetsFileChangeContext_FileChanged(object? sender, string filePath
         return _mostRecentFileInfo.TargetFramework;
     }
 
+    public ProjectSystemProject GetProjectSystemProject()
```

Comment:
It looks like caller is only using the project id. Let's expose just the project id.